server/bleep/src/indexes/file.rs

```diff
@@ -523,13 +538,13 @@ impl RepoDir {
                 schema.raw_repo_name => repo_name.as_bytes(),
                 schema.raw_relative_path => relative_path_str.as_bytes(),
                 schema.repo_disk_path => repo_disk_path.to_string_lossy().as_ref(),
-                schema.entry_disk_path => entry_pathbuf.to_string_lossy().as_ref(),
                 schema.relative_path => relative_path_str,
                 schema.repo_ref => repo_ref,
                 schema.repo_name => repo_name,
                 schema.last_commit_unix_seconds => last_commit,
                 schema.branches => branches,
                 schema.is_directory => true,
+                schema.unique_hash => tantivy_cache_key,
```

Comment:
Can you give this a more descriptive name?

Reply:
It's not intended to carry semantic meaning. It needs to fulfill 2 criteria:

 * db-wide unique
 * deterministic
 
Relying on or expressing any other incidental property these IDs may carry is not a good idea.